statistics/src/main/java/com/duckduckgo/app/statistics/api/OfflinePixelSender.kt

```diff
@@ -55,7 +54,10 @@ constructor(
                 sendCookieDatabaseOpenErrorPixel(),
                 sendCookieDatabaseDeleteErrorPixel(),
                 sendCookieDatabaseCorruptedErrorPixel(),
-                sendUncaughtExceptionsPixel()))
+                sendUncaughtExceptionsPixel(),
+                *offlinePixels.map { it.send() }.toTypedArray()
```

Comment:
I take the idea here will be to remove all the other methods that are called here and instead use the new `OfflinePixel` interface for all the other offline pixels?

Reply:
Yep, that's the idea. Will create the task for it